src/Button/OutlineButton.js

```diff
@@ -0,0 +1,22 @@
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import Button from './Button'
+
+const OutlineButton = styled(Button)`
+  background-color: ${props => props.theme.colors.white};
+  color: ${props => props.color || props.theme.colors.blue};
```

Comment:
You could use a default prop here instead of the `||`

Reply:
I'd also either use the styled-system `color` function or a function that can pick up values from `theme.colors` by key since that's how the Box component works. E.g. the prop `color='blue'` gets the color `theme.colors.blue`

Reply:
So maybe add `${color}` imported from `styled-system`?